systemd/ceph-mds@.service

```diff
@@ -11,14 +11,15 @@ EnvironmentFile=-/etc/sysconfig/ceph
 Environment=CLUSTER=ceph
 ExecStart=/usr/bin/ceph-mds -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
 ExecReload=/bin/kill -HUP $MAINPID
-PrivateDevices=yes
+PrivateDevices=no
```

Comment:
@Adirl, I believe @tchaikov is asking if it's feasible for you to add a file to your installation like `/etc/systemd/system/ceph-mds@.service.d/rdma.conf` containing:

````
[Service]
PrivateDevices=no
````

`PrivateDevices=yes` is an appropriate default for most installations.